src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -855,17 +855,19 @@ def getTrustSitelists(self):
         """
         _getTrustSitelists_
 
-        Accessor for the 'trust site lists' flag for the task.
+        Get the input and pileup flag for 'trust site lists' in the task.
         """
-        return self.data.constraints.sites.trustlists
+        return (self.data.constraints.sites.trustlists,
+                self.data.constraints.sites.trustPUlists)
```

Comment:
I agree! I made it a tuple to have a least "noisy" code, it's more straight forward, though not very clear.